scripts/src/java/org/oppia/android/scripts/XMLSyntaxCheck.kt

```diff
@@ -0,0 +1,71 @@
+package org.oppia.android.scripts
+
+import org.xml.sax.SAXParseException
+import java.io.File
+import javax.xml.parsers.DocumentBuilder
+import javax.xml.parsers.DocumentBuilderFactory
+
+/**
+ * Script for ensuring that all the XML files
+ * in the repo are syntactically correct.
+ */
+class XMLSyntaxCheck {
+  companion object {
+    @JvmStatic
+    fun main(vararg args: String) {
```

Comment:
For this & the regex check script: the main function should be documented as well. See my PR for an idea on how to do this--it's nice to document the actual usage of the script in the KDoc of main() since that's essentially how to "call" it.

Reply:
Done.